Dockerfile

```diff
@@ -10,6 +10,7 @@ MAINTAINER Peter Idah <peter.idah@digital.justice.gov.uk>
 
 # Set correct environment variables.
 ENV HOME /root
+ENV BACKEND_BASE_URI http://ip-172-31-27-7.eu-west-1.compute.internal
```

Comment:
Don't set these env vars in the docker file. It is already set from the deploy repo and should be the actual backend url not the internal address